src/libostree/ostree-repo-checkout.c

```diff
@@ -420,21 +378,12 @@ checkout_one_file_at (OstreeRepo                        *repo,
                       GCancellable                      *cancellable,
                       GError                           **error)
 {
-  gboolean ret = FALSE;
-  const char *checksum;
-  gboolean is_symlink;
-  gboolean is_bare_user_symlink = FALSE;
-  gboolean can_cache;
   gboolean need_copy = TRUE;
+  gboolean is_bare_user_symlink = FALSE;
   char loose_path_buf[_OSTREE_LOOSE_PATH_MAX];
-  g_autoptr(GInputStream) input = NULL;
-  g_autoptr(GVariant) xattrs = NULL;
-  gboolean is_whiteout;
-
-  is_symlink = g_file_info_get_file_type (source_info) == G_FILE_TYPE_SYMBOLIC_LINK;
-  checksum = ostree_repo_file_get_checksum ((OstreeRepoFile*)source);
-
-  is_whiteout = !is_symlink && options->process_whiteouts &&
+  const gboolean is_symlink = g_file_info_get_file_type (source_info) == G_FILE_TYPE_SYMBOLIC_LINK;
+  const char *checksum = ostree_repo_file_get_checksum ((OstreeRepoFile*)source);
+  const gboolean is_whiteout = !is_symlink && options->process_whiteouts &&
```

Comment:
Note this was just a moved hunk, not net new code.  But sure.  Fixups :arrow_down: